extensions/interactions/NumericExpressionInput/protractor.js

```diff
@@ -46,19 +48,19 @@ var submitAnswer = async function(elem, answer) {
 var answerObjectType = 'NumericExpression';
 
 var testSuite = [{
-  interactionArguments: [],
+  interactionArguments: ['Type an expression here, using only numbers.'],
```

Comment:
There's already a check for this here:
https://github.com/oppia/oppia/blob/615ac1b786cd4acdbb0e88b5c46697e8bbb6fc09/extensions/interactions/NumericExpressionInput/protractor.js#L33-L39

Am I missing something?

Reply:
This works, but it's a little strange because we pass an argument with `interactionArguments`, but then the check is hard-coded instead of being passed through `expectedInteractionDetails`:

https://github.com/oppia/oppia/blob/615ac1b786cd4acdbb0e88b5c46697e8bbb6fc09/extensions/interactions/NumericExpressionInput/protractor.js#L53

Can we pass the text `Type an expression here, using only numbers.` into the `expectInteractionDetailsToMatch` function through `expectedInteractionDetails`?

Once we have the expected text passed in through `expectedInteractionDetails`, we can test things like changing the message.

Reply:
Yeah, that makes sense. I've made changes, PTAL!

Reply:
Thanks! This looks great